Dockerfile.etl

```
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10.7
+WORKDIR /code
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# if you use an alpine python image, the following is necessary
+# RUN apk add --no-cache gcc musl-dev linux-headers
+# Note: This version of poetry must match what is in the `flake.nix`
+RUN pip install "poetry==1.3.2"
+
+RUN python -m venv /venv
+
+# Install dependencies first for better caching
+RUN mkdir -p ./build/api
+COPY api/pyproject.toml build/api/pyproject.toml
+COPY api/poetry.lock build/api/poetry.lock
```

Nowadays I see ADD being used more widely, we can do in one step which is faster
```suggestion
ADD ./api/pyproject.toml ./api/poetry.lock build/api/
```